src/main.tsx

```diff
@@ -0,0 +1,11 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import "./index.css";
+import App from "./App.tsx";
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+createRoot(document.getElementById("root")!).render(
```

Comment:
Not sure if this is a Vite thing or a React thing, but typescript-eslint doesn't like the [non-null-assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-) here. For this case I think it is fine to have, and removing it is more work than it's worth, so I think it's fine to ignore the linter here.

This may be causing a problem because I turned strict checks on for tseslint, but we can always turn those off if it becomes a problem down the line.

Reply:
👍